docs/openapi/openapi.yml

```diff
@@ -25,5 +21,5 @@ components:
       scheme: bearer
 
 paths:
-  /oauth/token:
+  /api/v1/oauth/token:
```

Comment:
Can we keep the previous version with [Server Templating](https://swagger.io/docs/specification/api-host-and-base-path/) `url: https://{base_url}/api/v1`?
IMO, if we need to define the `v2` API, we should create another file, right?

Reply:
Nice catch! We could do that

Reply:
@hoangmirs Fixed in https://github.com/nimblehq/nimble-survey-web/pull/88/commits/4a8070f6fe4ebb1db08d46b104ab07dd5d8dbfff

Please note that I'm not using [server templating](https://swagger.io/docs/specification/api-host-and-base-path/#:~:text=url%3A%20/-,Server%20Templating,-Any%20part%20of). Define multiple server URLs supports switching easily